doc/src/pair_hybrid.rst

```diff
@@ -100,6 +100,52 @@ first is assigned to intra-molecular interactions (i.e. both atoms
 have the same molecule ID), the second to inter-molecular interactions
 (i.e. interacting atoms have different molecule IDs).
 
+.. admonition:: When **NOT** to use a hybrid pair style
+   :class: warning
+
+   Using pair style *hybrid* can be very tempting to use if you need a
+   **many-body potential** supporting a mix of elements for which you
+   cannot find a potential file that covers *all* of them.  Regardless
+   of how this is set up, there will be *errors*.  The major use case
+   where the error is *small*, is when the many-body sub-styles are used
+   on different objects (for example a slab and a liquid, a metal and a
+   nano-machining work piece).  In that case the *mixed* terms **must**
```

Comment:
"must" implies that LAMMPS will throw an error.  I don't think that is the case ?
I think you mean "should" (to minimize errors).  LAMMPS will allow you to do otherwise
but that way leads to darkness ...